packages/helpers/src/deployer.js

```diff
@@ -19,6 +19,22 @@ async function deployTimelockManager(deployer, ownerAddress, api3TokenAddress) {
   return timelockManager;
 }
 
+async function deployTimelockManagerReversible(
```

Comment:
I just added the deployer script, you can have a look. Its using the DAO address for the owner, since there wasn't any other address available in the deployments.json that I think was appropriate.